MonoGame.Framework/SDL/SDLGameWindow.cs

```diff
@@ -236,8 +236,11 @@ public override void EndScreenDeviceChange(string screenDeviceName, int clientWi
                 _height = displayRect.Height;
             }
 
-            var centerX = Math.Max(prevBounds.X + ((prevBounds.Width - clientWidth) / 2), 0);
-            var centerY = Math.Max(prevBounds.Y + ((prevBounds.Height - clientHeight) / 2), 0);
+            int ignore, minx = 0, miny = 0;
+            Sdl.Window.GetBorderSize(_handle, out miny, out minx, out ignore, out ignore);
+
+            var centerX = Math.Max(prevBounds.X + ((prevBounds.Width - clientWidth) / 2), minx);
+            var centerY = Math.Max(prevBounds.Y + ((prevBounds.Height - clientHeight) / 2), miny);
```

Comment:
So if i read this right... you're just adjusting the center so that the title is visible and the bottom of the window is offscreen.  The window size isn't changing at all.  Correct?

Reply:
I just changed the minimum possible value from `0` to top and left border sizes.